usefulUtilities/hiFiCalendar/entities/calendarEntities.json

```diff
@@ -2,34 +2,36 @@
   "DataVersion": 0,
   "Entities": [
     {
-    "id": "{30de79ea-a40b-499b-83c6-06dbfcf9febb}",
+    "id": "{455f56c8-9a76-420f-a8ab-6dad5da7358a}",
     "type": "Text",
-    "lastEdited": 1559239932550773,
-    "parentID": "{a81d9ae5-6df9-4d2d-9d40-4062752f1973}",
-    "name": "Calendar_MR1_Main_RoomColorOccupant",
+    "lastEdited": 1561571024131146,
+    "parentID": "{2175cd3b-9bfd-4892-b666-98d50a975f6d}",
+    "name": "Calendar_MR1_Main_RoomOccupantsList",
+    "userData": "{\"roomOccupantsListID\":\"\",\"meetingZoneID\":\"\"}",
+    "description": "CFGBLE -",
```

Comment:
is this actually configurable?

Reply:
This one needs the id of the meeting zone.  Does that qualify it for the configurable label?

Reply:
yes